README.md

```diff
@@ -5,6 +5,19 @@ Central UI server connecting to other Passport-related microservices (ory hydra,
 This service is a docker container, having [Docker Desktop](https://docs.docker.com/desktop/) installed beforehand will be useful. This repo also uses a [Makefile](./Makefile), which allows defining special shell commands. If you would like to see what does the make commands do in more detail, please visit the file.
 
 ***
+###### Install Node and NPM
+Visit [Node.js's official website](https://nodejs.org/en/download/) to download the framework. 
```

Comment:
Could you please add the node and npm versions that need to be installed? You can simply add the versions installed on your local computer.

This is important because if in the future there are any breaking changes to node/npm, we should make sure we are using the correct version to build our code.

Reply:
versions added now